Dockerfile

```diff
@@ -100,6 +100,9 @@ ENV IMAGE_NAME=saucelabs/stt-playwright-node
 ARG BUILD_TAG
 ENV IMAGE_TAG=${BUILD_TAG}
 
+# Let saucectl know where to read job details url
+LABEL com.saucelabs.job-details-url=/tmp/output-job-details-url
```

Comment:
Just a food for thought, but do we potentially want this file to be a json file? That way, should the need ever arise, it'd be a bit more extensible, in case we want to transmit other information 🤷 But, not necessary.

Reply:
@alexplischke Yes, it makes more sense, so we have a easy method for running data transmission from containers to saucectl.
As it will be more complicated to implement it later (due to potential diversity of saucectl in the wild), it's better to do it now IMO (and this is really quick to implement).

Reply:
@alexplischke  this is implemented now :)